tests/other_test.go

```diff
@@ -33,6 +32,21 @@ func TestRunCompleteExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunExistingSecretManagerExample(t *testing.T) {
```

Comment:
Instead of adding a new test, can you just update the `TestRunExistingResourcesInstances` test and pass existing SM instance to that

Reply:
I don't think you need a new example either